src/app/components/authenticated-v2/components/item-icon/item-icon.component.ts

```diff
@@ -1,26 +1,23 @@
-import {AfterViewInit, Component, Input, OnInit} from '@angular/core';
-import {IManifestArmor} from "../../../../data/types/IManifestArmor";
-import {ItemIconServiceService} from "../../../../services/item-icon-service.service";
+import { AfterViewInit, Component, Input, OnInit } from "@angular/core";
+import { IManifestArmor } from "../../../../data/types/IManifestArmor";
+import { ItemIconServiceService } from "../../../../services/item-icon-service.service";
 
 @Component({
-  selector: 'item-icon',
-  templateUrl: './item-icon.component.html',
-  styleUrls: ['./item-icon.component.scss']
+    selector: "app-item-icon",
```

Comment:
The default settings for angualr linters enforces the use of "app-" in your own components.
I changed it for this one, and then decided to just disable it because I did not like it. Our own internal components will never be shared outside of this project.

Reply:
Alright, wanna change it back here too or is it fine?